02-peer-protocol.md

```diff
@@ -30,8 +30,8 @@ operation, and closing.
 
 ## Channel Establishment
 
-Channel establishment begins immediately after authentication and
-consists of the funding node (funder) sending an `open_channel` message,
+After authentication and initializing a connection, channel establishment may begin.
```

Comment:
I guess it SHOULD begin. Right? We want to promote liveness of channels.

Reply:
I think there are many good cases to have a connection in gossiping state. Random connection to learn the topology for example.

Reply:
Nvm, thought this was re-establish!